README.md

```diff
@@ -131,8 +131,8 @@ need them.
 
 Rustls uses [`ring`](https://crates.io/crates/ring) for implementing the
 cryptography in TLS. As a result, rustls only runs on platforms
-[supported by `ring`](https://github.com/briansmith/ring#online-automated-testing).
-At the time of writing this means x86, x86-64, armv7, and aarch64.
+supported by `ring`. At the time of writing this means x86, x86-64, armv7, and
```

Comment:
That is too vague, as I wrote in https://github.com/rustls/rustls/issues/1275#issuecomment-1507370554, ARMv7 for example is incredibly vague (hard float? Linux, Windows or FreeBSD? Bare metal?). And Aarch64 is available in a big endian variant (apparently, TIL). You need a list of specific triplets that are supported, or at least globs of triplets.

Reply:
I've added a link to the [Ring CI configuration](https://github.com/briansmith/ring/blob/9cc0d45f4d8521f467bb3a621e74b1535e118188/.github/workflows/ci.yml#L151-L167) where the supported triplets are listed.